### PR TITLE
fix: use tutor variables to expose clickhouse container ports in k8s

### PR DESCRIPTION
**Pull request by pomegranited**
### Description

Updates the k8s configuration to use Tutor variables when exposing the Clickhouse container ports.

### Related information

Closes https://github.com/openedx/tutor-contrib-aspects/issues/387

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutoraspects/patches/k8s-deployments [hunk 1]
```diff
@@ -30,9 +30,9 @@ spec:
           image: {{ DOCKER_IMAGE_CLICKHOUSE }}
           name: clickhouse
           ports:
-            - containerPort: 8123
-            - containerPort: 9000
-            - containerPort: 9009
+            - containerPort: "{{ CLICKHOUSE_HOST_INSECURE_HTTP_PORT }}"
+            - containerPort: "{{ CLICKHOUSE_HOST_INSECURE_NATIVE_PORT }}"
+            - containerPort: "{{ CLICKHOUSE_HOST_TLS_NATIVE_PORT }}"
```
Comment (pomegranited) [inline tutoraspects/patches/k8s-deployments:35]: Note: the default values for these variables differ from what was hardcoded here, so operators may need to update their configurations.

The default values are:

* [`CLICKHOUSE_HOST_INSECURE_HTTP_PORT: 8123`](https://github.com/openedx/tutor-contrib-aspects/blob/3d6399109d73c4b87fafb1bd0d145f3407cf4971/tutoraspects/plugin.py#L144) (unchanged)
*  [`CLICKHOUSE_HOST_INSECURE_NATIVE_PORT: 9006`](https://github.com/openedx/tutor-contrib-aspects/blob/3d6399109d73c4b87fafb1bd0d145f3407cf4971/tutoraspects/plugin.py#L117) (changed from 9000)
* [`CLICKHOUSE_HOST_TLS_NATIVE_PORT: 9440`](https://github.com/openedx/tutor-contrib-aspects/blob/3d6399109d73c4b87fafb1bd0d145f3407cf4971/tutoraspects/plugin.py#L123) (changed from 9006)

**Review by Ian2012 — APPROVED** [review]

File: tutoraspects/patches/k8s-deployments (hunk 1 above)
Comment (Ian2012) [reply tutoraspects/patches/k8s-deployments]: We are running Aspects with Clickhouse outside the cluster. I think it's safe to make the change

**Comment by openedx-webhooks** [comment]
@pomegranited 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/524&entry.1484655318=https://github.com/pomegranited&entry.752974735=2023-11-22+04:07&entry.1917517419=2023-11-23+03:21&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**